xml/test/application-concurrency.xml

```diff
@@ -34,17 +34,23 @@
     <cleared></cleared>
     <propagated>Security</propagated>
     <unchanged>Remaining</unchanged>
+    <qualifier>ee.jakarta.MyQualifier</qualifier>
   </context-service>
 
   <managed-executor>
     <name>java:app/concurrent/Executor20</name>
     <context-service-ref>java:app/concurrent/SecContextOnly</context-service-ref>
     <max-async>20</max-async>
+    <qualifier>ee.jakarta.MyQualifier</qualifier>
+    <qualifier>ee.jakarta.MyQualifier2</qualifier>
```

Comment:
The ordering of the elements in the application_11.xsd for the managed-executorType has the virtual element before the qualifier element, which is why you are seeing validation errors.